packages/perseus/src/widgets/group/group.testdata.ts

```diff
@@ -185,3 +191,38 @@ export const simpleGroupQuestion: PerseusRenderer = {
         },
     },
 };
+
+export function getFullGroupTestItem(): PerseusItem {
+    const groupRenderer = generateTestPerseusRenderer({
+        content: "Group Renderer\n\n[[☃ dropdown 1]]",
+        widgets: {
+            "dropdown 1": {
+                type: "dropdown",
+                options: {
+                    choices: [
+                        {content: "Incorrect", correct: false},
+                        {content: "Correct", correct: true},
+                    ],
+                    placeholder: "Choose an answer",
+                    static: false,
+                },
+            },
+        },
+    });
+
+    const itemRenderer = generateTestPerseusRenderer({
+        content: "Item Renderer\n\n[[☃ group 1]]",
+        widgets: {
+            "group 1": {
+                type: "group",
+                options: groupRenderer,
+            },
+        },
+    });
+
+    return generateTestPerseusItem({question: itemRenderer});
+}
+
+export function getSplitGroupTestItem(): PerseusItem {
```

Comment:
"Split" means "answerless" and "full" means "answerful", right?

I wonder if these exports should be const values rather than functions.